src/com/videojs/providers/HTTPAudioProvider.as

```diff
@@ -134,6 +134,18 @@ package com.videojs.providers{
             return [];
         }
 
+        /** HTTPAudioProvider does not support seekableStart
```

Comment:
sorry for the nit-picky, but can you fix the formatting on these comments (also in the other providers)
```
/**
 * HTTPAudioProvider does not support seekableStart
 */
```